core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -1652,16 +1669,14 @@ protected boolean hasValidFileAndUrl(String url, ZimFileReader zimFileReader) {
   @Override
   public void webViewFailedLoading(String url) {
     String error = String.format(getString(R.string.error_article_url_not_found), url);
-    Toast.makeText(this, error, Toast.LENGTH_SHORT).show();
+    Toast.makeText(getActivity(), error, Toast.LENGTH_SHORT).show();
   }
 
   @Override
   public void webViewProgressChanged(int progress) {
     if (checkNull(progressBar)) {
-      progressBar.show();
```

Comment:
these lines were deleted?

Reply:
Very odd. I don't remember deleting this..